src/ExportHistory/Models/ExportJobCreationOptions.cs

```diff
@@ -0,0 +1,178 @@
+        {
+            if (completedTimeFrom != default)
+            {
+                throw new ArgumentException(
+                    "CompletedTimeFrom is not allowed for Continuous export mode.",
+                    nameof(completedTimeFrom));
+            }
+
+            if (completedTimeTo.HasValue)
+            {
+                throw new ArgumentException(
+                    "CompletedTimeTo is not allowed for Continuous export mode.",
+                    nameof(completedTimeTo));
+            }
+        }
+        else
+        {
+            throw new ArgumentException(
+                "Invalid export mode.",
+                nameof(mode));
+        }
+
+        // Validate maxInstancesPerBatch range if provided (must be 1..999)
+        if (maxInstancesPerBatch.HasValue && (maxInstancesPerBatch.Value <= 0 || maxInstancesPerBatch.Value >= 1001))
+        {
+            throw new ArgumentOutOfRangeException(
+                nameof(maxInstancesPerBatch),
+                maxInstancesPerBatch,
+                "MaxInstancesPerBatch must be between 1 and 1000.");
+        }
+
+        // Validate terminal status-only filter here if provided
+        if (runtimeStatus is { Count: > 0 }
+            && runtimeStatus.Any(
+                s => s is not (OrchestrationRuntimeStatus.Completed
+                               or OrchestrationRuntimeStatus.Failed
+                               or OrchestrationRuntimeStatus.Terminated
+                               or OrchestrationRuntimeStatus.ContinuedAsNew)))
+        {
+            throw new ArgumentException(
+                "Export supports terminal orchestration statuses only. Valid statuses are: Completed, Failed, Terminated, and ContinuedAsNew.",
+                nameof(runtimeStatus));
+        }
+
+        this.Mode = mode;
+        this.CompletedTimeFrom = completedTimeFrom;
+        this.CompletedTimeTo = completedTimeTo;
+        this.Destination = destination;
+        this.Format = format ?? ExportFormat.Default;
+        this.RuntimeStatus = (runtimeStatus is { Count: > 0 })
+            ? runtimeStatus
+            : new List<OrchestrationRuntimeStatus>
+            {
+                OrchestrationRuntimeStatus.Completed,
+                OrchestrationRuntimeStatus.Failed,
+                OrchestrationRuntimeStatus.Terminated,
+                OrchestrationRuntimeStatus.ContinuedAsNew,
+            };
+        this.MaxInstancesPerBatch = maxInstancesPerBatch ?? 100;
+    }
+
+    /// <summary>
+    /// Gets the unique identifier for the export job.
+    /// </summary>
+    public string JobId { get; init; }
+
+    /// <summary>
+    /// Gets the export mode (Batch or Continuous).
+    /// </summary>
+    public ExportMode Mode { get; init; }
+
+    /// <summary>
+    /// Gets the start time for the export based on completion time (inclusive).
+    /// Required for Batch mode. For Continuous mode, this is automatically set to UtcNow when creating the job.
```

Comment:
The code requires it to be null, could be worth calling out here, but I also don't think we have to avoid this. They could want to start a continuous flow but also include the previous x minutes/hours.